docs/manual/developer_guide.adoc

```diff
@@ -240,7 +240,7 @@ The prose guide HTML contains practical, actionable information for auditors
 and administrators.
 
 Spreadsheet HTML tables - potentially useful as the basis for an
-*SRTM document*:
+*SRTM (Security Requirements Traceability Matrix) document*:
```

Comment:
Let's flip this to `Security Requirements Traceability Matrix (SRTM)`

Reply:
:+1: